static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -1,11 +1,301 @@
+        border-top: 1px solid $media-mute;
+        padding-top: $gs-baseline / 4;
+        display: block;
+        width: gs-span(1);
+    }
+}
+
+.gallery__divider {
+    border-top: 1px solid $media-mute;
+    margin-top: $gs-baseline;
+
+    @include mq(desktop) {
+        padding-top: $gs-baseline / 2;
+    }
+}
+
+.content__main-column--gallery {
+    max-width: none;
+    margin-right: auto;
+    padding-bottom: $gs-baseline*2;
+
+    @include mq(desktop) {
+        margin-left: gs-span(3) + $gs-gutter;
+        margin-right: -$gs-gutter;
+    }
+}
+
+.gallery {
+    list-style: none;
+    margin: 0;
+    position: relative;
+
+    @include mq(desktop) {
+        &:before {
+            content: '';
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            left: -$gs-gutter/2;
+            border-left: 1px solid $media-mute;
+        }
+    }
+}
+
+.gallery__item {
+    figure {
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+    // On mobile/tablet the images should be 100% of the width of the screen
+    @include mq($until: desktop) {
+        margin-left: -$gs-gutter;
+        margin-right: -$gs-gutter;
+        padding-top: $gs-baseline / 2;
+        border-top: 1px solid $media-mute;
+
+        &:first-of-type {
+            border-top: 0;
+        }
+    }
+
+    @include mq($until: mobileLandscape) {
+        margin-left: -$gs-gutter / 2;
+        margin-right: -$gs-gutter / 2;
+    }
+
+    @include mq(desktop) {
+        position: relative;
+        border-top: 0;
+        padding: 0;
+
+        figure {
+            flex-direction: row;
+        }
+    }
+}
+
+.gallery__item--advert {
+    @include mq($until: desktop) {
+        border-top: 0;
+        margin-left: 0;
+        margin-right: 0;
+    }
+}
+
+.gallery__figcaption {
+    @include fs-textSans(1);
+    overflow: visible;
+    max-height: 100%;
+    position: relative;
+    border: 0;
+    width: inherit;
+
+    @include mq($until: desktop) {
+        margin-bottom: $gs-baseline * 2;
+        padding-top: $gs-baseline / 2;
+        padding-left: $gs-gutter;
+        padding-right: $gs-gutter;
+    }
+
+    @include mq($until: mobileLandscape) {
+        padding-left: $gs-gutter / 2;
+        padding-right: $gs-gutter / 2;
+    }
+
+    @include mq(desktop) {
+        @include fs-textSans(2, true);
+        position: absolute;
+        width: gs-span(3);
+        top: 0;
+        right: auto;
+        margin-left: -(gs-span(3) + $gs-gutter);
+    }
+}
+
+.gallery__caption {
+    color: $neutral-4;
+
+    @include mq(desktop) {
+        padding-right: $gs-gutter / 2;
+    }
+
+    a {
+        color: $neutral-4;
+        border-bottom: 1px solid $neutral-2;
+        transition: border-color .15s ease-out;
+
+        &:hover {
+            border-bottom: 1px solid $neutral-7;
+            text-decoration: none;
+        }
+    }
+
+    // This is the gallery caption title
+    > strong:first-child {
```

Comment:
Yes I think you are right and we can add another cleaner to do this as well. I am happy to do that, but I am unsure which approach is better. It seems to be that this would be simply moving the complexity.